projects/helm/helm/patches/0001-Insecure-flag-for-oci-push-pull-install.patch

```diff
@@ -5,25 +5,12 @@ Subject: [PATCH 1/2] Push to insecure OCI registry
 
 Signed-off-by: acool@amazon.com
 ---
- cmd/helm/pull.go       |  1 +
  cmd/helm/push.go       |  4 ++++
  pkg/action/pull.go     |  7 +++++++
  pkg/action/push.go     | 12 ++++++++++--
  pkg/registry/client.go | 31 +++++++++++++++++++++++++++++++
- 5 files changed, 53 insertions(+), 2 deletions(-)
+ 4 files changed, 52 insertions(+), 2 deletions(-)
 
-diff --git a/cmd/helm/pull.go b/cmd/helm/pull.go
```

Comment:
Was this part merged upstream?

Reply:
This patch was essentially a no-op, it's being added in one patch and removed in the other. In the second patch, the `plain-http` option is being added in the common flags definition [flags.go](https://github.com/aws/eks-anywhere-build-tooling/pull/1079/files#diff-4d717bde97d6fc33b98147beb5ade9ea748cfc72be07f22bb162d6da57288904R152), as opposed to for each Helm command like push/pull.